airflow/utils/db.py

```diff
@@ -280,6 +280,14 @@ def initdb(rbac=False):
         Connection(
             conn_id='azure_cosmos_default', conn_type='azure_cosmos',
             extra='{"database_name": "<DATABASE_NAME>", "collection_name": "<COLLECTION_NAME>" }'))
+    merge_conn(
```

Comment:
Why you need create a seperate connection per service? Why is connection `azure default` not right?

Reply:
All other Azure services have their own connection and I went with the convention. Do you recommend I change all of them to "azure default" or only mine?

Reply:
I am suggesting here to use `azure_default`, and then prepare another PR that will introduce changes to other operators. 
Look from the user's perspective. Would you like to give the same password several times? It will not be comfortable.

Reply:
Hi @mik-laj , i have this same problem. Because Azure usually uses ```ServicePrincipalCredentials``` and other optional auth(like DB name or rest username/pass). 
My suggestion is creating uniform connection like GCP and it should only include ```ServicePrincipalCredentials``` from auth file, Other credentials(like username/pass for optional) should be in extras.
That way we can use ```azure_default ```.

Reply:
Okay, thanks for clarifying. I will change that.